dev-java/miglayout/miglayout-4.0-r1.ebuild

```diff
@@ -20,11 +20,11 @@ COMMON_DEPEND="swt? ( dev-java/swt:3.7 )"
 BDEPEND="app-arch/unzip"
 RDEPEND="
 	${COMMON_DEPEND}
-	>=virtual/jre-1.5
+	>=virtual/jre-1.8:*
 "
 DEPEND="
 	${COMMON_DEPEND}
-	>=virtual/jdk-1.5
+	>=virtual/jdk-1.8:*
```

Comment:
When fixing open bugs please don't forget to have them mentioned in the commit message: https://bugs.gentoo.org/786732

Reply:
@vaukai I'll get in the habit of checking for open bugs as I work through this list.

I think I've seen some talk of some sort of cli interface to bugzilla, this will be a good excuse for me to check that out.

Also, I will go back and check all the commit messages so far too.

Reply:
Ok, I've gone back and checked for bugs for each commit in this PR and updated the commit messages as appropriate. Closing this as resolved.

Reply:
On https://bugs.gentoo.org/786732 I don't see the PR.
```
In order to force reassignment and/or bug reference scan, please append [please reassign] to the pull request title.
```